src/groupeddataframe/grouping.jl

```diff
@@ -548,8 +551,6 @@ faster than returning a data frame.
 As a special case, if a tuple or vector of pairs is passed as the first argument, each function
 is required to return a single value or vector, which will produce each a separate column.
 
-In all cases, the resulting `GroupedDataFrame` contains all the grouping columns in addition
```

Comment:
I think this sentence is also used for `combine`: maybe apply the same move there?

Reply:
OK - I will sync them when we settle the comment I have left above regarding the return value.